src/ert/storage/local_experiment.py

```diff
@@ -147,10 +148,6 @@ def __init__(  # pylint: disable=too-many-arguments
         with open(response_file, "w", encoding="utf-8") as f:
             json.dump(response_data, f, default=str)
 
-    @property
-    def ensembles(self) -> Generator[LocalEnsembleAccessor, None, None]:
-        yield from super().ensembles  # type: ignore
-
```

Comment:
why is this function not needed?

Reply:
My thinking is that it's not needed since it just calls its parents `ensembles`.

This also makes the typing misleading as the parent is a `LocalExperiment` reader and its `ensemble` property has the signature: `def ensembles(self) -> Generator[LocalEnsembleReader, None, None]:`.

Reply:
Okay, I checked where `ensembles()` function is called, and to me it looks like indeed only parent function is used 👍